src/auto.ts

```diff
@@ -492,9 +495,39 @@ export default class Auto {
     this.logger.verbose.info("Using command: 'shipit'");
     this.hooks.beforeShipIt.call();
 
+    const isPR = 'isPr' in env && env.isPr;
+    // env-ci sets branch to target branch (ex: master) in some CI services.
+    // so we should make sure we aren't in a PR just to be safe
+    const isMaster = !isPR && 'branch' in env && env.branch === 'master';
```

Comment:
Probably wouldn't be too hard to go ahead and make this `isBaseBranch` and set a `BASE_BRANCH` const somewhere to compare it to. We can make it configurable in a separate PR.

Reply:
I have made the base branch configurable in this PR https://github.com/intuit/auto/pull/357.